lib/src/blocs/contract/i_bloc.dart

```diff
@@ -0,0 +1,5 @@
+abstract class IBloc {
+  void initialize();
```

Comment:
Initialize could have async work, so lets use this interface as `Future<void>`

Reply:
Done